praw/__init__.py

```diff
@@ -2300,6 +2300,50 @@ def send_message(self, recipient, subject, message, from_sr=None,
         return response
 
 
+class ReportMixin(AuthenticatedReddit):
+
+    """Adds methods requiring the 'report' scope (or login).
+
+    You should **not** directly instantiate instances of this class. Use
+    :class:`.Reddit` instead.
+
+    """
+
+    @decorators.restrict_access(scope='report')
+    def hide(self, thing_id, unhide=False):
```

Comment:
Can you prefix `unhide` with an underscore as `_unhide` to indicate that it should _not_ be used directly in this way so we are free to change the implementation to remove that parameter if we want.